### DISCUSSION TITLE
日志重复采集问题【急】，谢谢~

### DISCUSSION THREAD

**Discussion by yecun123** [alibaba/ilogtail#960] [Help]
问题阐述：两台部署同一个服务（check）所在的容器节点，其它微服务包括该微服务日志出现了重复采集，而且是多次重复采集。
现象：check微服务日志量是较大的，一天大概40-50个G左右，日志发生切割时，会压缩成压缩包（目前不确定是否因该原因引起），Ilogtail日志上有看到有采集延迟，CPU会偶现突增现象。
日志重采截图：
![image](https://github.com/alibaba/ilogtail/assets/125344638/111053bf-2d54-482d-98c0-0dc79cbe7fc9)
ilogtail采集配置:
ilogtail-config-json: 
    {
        "polling_ignore_file_modify_timeout": 7200,
        "polling_file_first_watch_timeout": 600,
        "check_point_dump_interval": 5,
        "cpu_usage_limit": 5,
        "mem_usage_limit": 1024
    }
节点CPU使用率:
节点日志较大时，CPU突增
![image](https://github.com/alibaba/ilogtail/assets/125344638/a0f94454-3480-4e9a-b8da-b180c45c7869)
Ilogtail出现重采发现异常日志：
[ilogtail.LOG](https://github.com/alibaba/ilogtail/files/11913810/ilogtail.LOG)
望及时回复，谢谢~

**Comment by messixukejia** [comment]
需要提供下插件日志 跟 采集配置（yaml文件）。
